nishimweelysee/Elysee-CapstoneProject
# Ft admin page

#### What does this PR do?
> - This PR contain an admin page that admin use to monitor and manager all activities from a webpages.
#### Description of Task to be completed
>-Blog Tab where user update,create, list and delete blogs
>-Portfolio Tab where user update, create, list and delete a portfolio
>-Contacts Tab where a user display and answer to all questions or comments
>-Other Option tab where an admin can view the location of users, get Number of all Entries made and reminder setting
>
#### How should this be manually tested?
> Right now admin button is on the navigation, but after having account the user who is admin will only see the admin button. when he click on that button he will see the page where he can monitor all actions.
#### Any background context you want to provide?
> this is a page to manage all action from the website.
#### What are the relevant Trello story?
> Ft( Admin Page) Use this link https://trello.com/c/bZ73lhWx to visi my trello board.
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92327578-e3ee7580-f05a-11ea-821f-f8d6a2c01b10.png)

Thank You !!
